scripts/run_lighthouse_tests.py

```diff
@@ -139,22 +186,33 @@ def start_google_app_engine_server():
     SUBPROCESSES.append(p)
 
 
-def main():
+def main(args=None):
     """Runs lighthouse checks and deletes reports."""
+    parsed_args = _PARSER.parse_args(args=args)
+
+    if not (parsed_args.accessibility or parsed_args.performance):
```

Comment:
Make sure both aren't specified simultaneously, too.

Perhaps better to have just one arg, with a limited range of choices? E.g.  `--mode=accessibility` or `--mode=performance`

Reply:
Done. Have switched to using one arg

**No Flags**
```
(env) (base) Jims-MBP:oppia jimzhan$ python -m scripts.run_lighthouse_tests
usage: run_lighthouse_tests.py [-h] --mode {accessibility,performance}
run_lighthouse_tests.py: error: argument --mode is required
```
**Both Flags**
```
(env) (base) Jims-MBP:oppia jimzhan$ python -m scripts.run_lighthouse_tests --mode accessibility performance
usage: run_lighthouse_tests.py [-h] --mode {accessibility,performance}
run_lighthouse_tests.py: error: unrecognized arguments: performance
(env) (base) Jims-MBP:oppia jimzhan$ 
```
**Only --mode**
```
(env) (base) Jims-MBP:oppia jimzhan$ python -m scripts.run_lighthouse_tests --mode
usage: run_lighthouse_tests.py [-h] --mode {accessibility,performance}
run_lighthouse_tests.py: error: argument --mode: expected one argument
```
**h flag**
```
(env) (base) Jims-MBP:oppia jimzhan$ python -m scripts.run_lighthouse_tests -h
usage: run_lighthouse_tests.py [-h] --mode {accessibility,performance}

Run the script from the oppia root folder: python -m
scripts.run_lighthouse_tests Note that the root folder MUST be named 'oppia'.
optional arguments:
  -h, --help            show this help message and exit
  --mode {accessibility,performance}
                        Sets the mode for the lighthouse tests
(env) (base) Jims-MBP:oppia jimzhan$ 
```